src/LazyOperations/Complement.jl

```diff
@@ -38,8 +38,15 @@ struct Complement{N<:Real, S<:LazySet{N}}
 end
 
 isoperationtype(::Type{<:Complement}) = true
+
+# the set complement is not convex in general
 isconvextype(::Type{<:Complement}) = false
 
+# special cases which are always convex
+isconvextype(::Type{Complement{<:HalfSpace}}) = true
```

Comment:
This does not work. The first type parameter is the number type. Please add tests to avoid this.

Reply:
Good catch, thanks.